idaes/gas_solid_contactors/properties/methane_iron_OC_reduction/tests/test_CLC_solid_prop.py

```diff
@@ -203,4 +216,25 @@ def test_solution(solid_prop):
 
 @pytest.mark.component
 def test_units_consistent(solid_prop):
+
+    # Construct property methods to build the constraints
+
+    assert hasattr(solid_prop.fs.unit, "flow_mass")
+    assert hasattr(solid_prop.fs.unit, "particle_porosity")
+    assert hasattr(solid_prop.fs.unit, "temperature")
+    assert hasattr(solid_prop.fs.unit, "mass_frac_comp")
```

Comment:
These can be removed as they are state variables, so there are not property methods associated for them.